pkg/api/volume_req.go

```diff
@@ -137,6 +137,20 @@ type VolEditReq struct {
 	DeleteMetadata bool              `json:"delete-metadata"`
 }
 
+// ReplaceBrickReq represents replace brick request
+type ReplaceBrickReq struct {
+	SrcPeerID    string          `json:"srcpeerid"`
+	SrcBrickPath string          `json:"srcbrickpath"`
+	NewPeerID    string          `json:"newpeerid,omitempty"`
+	NewBrickPath string          `json:"newbrickpath,omitempty"`
```

Comment:
Do not accept new Brick path. It should remain same as existing path if possible else generate with the same format as of other bricks. Accepting brick path may not match with other bricks naming conventions